JoinColony/colonyCDapp
# [Verified members] Fix core UI issues

## Description

This PR fixes two UI issues:

Verified Icon in the verified members action table was 20px and had 8px left padding - this PR updates the sizing of the icon to 14px and the left padding to 4px aligning with figma

Table row meatball item in a hover state breaks the UI but this was only occuring in firefox. 

## Testing

* Go to verified members action
*  select 'Remove members'
* Add a member to the selected members table
* Verify the verified icon size is 14px with 4px left padding

Firefox is required to test the meatball item issue
* select the table row meatball
* hover over 'Remove Row' and verify that it's hover state doesn't break the UI

**Changes** 🏗


Hovering over the 'remove account' item in firefox:
<img width="684" alt="Screenshot 2024-03-12 at 11 40 08" src="https://github.com/JoinColony/colonyCDapp/assets/155957480/c7230645-817d-49a0-9da8-bb313636a007">

Icon sizing updated:
<img width="166" alt="Screenshot 2024-03-12 at 11 40 38" src="https://github.com/JoinColony/colonyCDapp/assets/155957480/39ada0a1-7465-402e-8994-2ea4abc617a6">



Resolves #2029
